bazel/test_external/WORKSPACE

```diff
@@ -10,3 +10,7 @@ deps()
 load("@emsdk//:emscripten_deps.bzl", "emscripten_deps")
 
 emscripten_deps()
+
+load("@emsdk//:toolchains.bzl", "register_emscripten_toolchains")
+
+register_emscripten_toolchains()
```

Comment:
Can/should we fold these two into a single step?

Reply:
Seems like other rulesets tend to keep them separate. E.g. rules_go has `go_rules_dependencies ` and `go_register_toolchains `, rules_nodejs has `build_bazel_rules_nodejs_dependencies` and `node_repositories`, rules_rust has `rules_rust_dependencies` and `rust_register_toolchains`.

Reply:
I saw that #1068 spells this `emsdk_register_toolchains`. Based on other rulesets, it does seem like the prevailing convention is to prefix with the language or toolset, rather than the verb `register`. However, I would propose that we retain consistency with `emscripten_deps`, and therefore spell this `emscripten_register_toolchains`. @walkingeyerobot @jun-sheaf WDYT?

Reply:
Hey @jfirebaugh! Your observation is correct. But just some information regarding this, I have upstreamed this to internal already. Hopefully by August I can get the internal toolchain open-sourced.

Reply:
What does "upstreamed to internal" mean?

Reply:
I meant the patch has already been published within Google (the internal repo).